docs/modules/druid/pages/usage-guide/operations/graceful-shutdown.adoc

```diff
@@ -1,7 +1,179 @@
 = Graceful shutdown
 
-Graceful shutdown of Druid nodes is either not supported by the product itself
-or we have not implemented it yet.
+You can configure the graceful shutdown as described in xref:concepts:operations/graceful_shutdown.adoc[].
 
-Outstanding implementation work for the graceful shutdowns of all products where this functionality is relevant is tracked in
-https://github.com/stackabletech/issues/issues/357
+The Druid processes will receive a `SIGTERM` signal when Kubernetes wants to terminate the Pod.
+It will log the received signal as shown in the log below and initiate a graceful shutdown.
+After the graceful shutdown timeout runs out, and the process still didn't exit, Kubernetes will issue a `SIGKILL` signal.
+
+== Broker
+
+As a default, Druid brokers have `5 minutes` to shut down gracefully.
+
+They use the same mechanism described above.
+
+[source,text]
+----
+druid 2023-11-13T10:47:13,194 INFO [Thread-55] org.apache.druid.java.util.common.lifecycle.Lifecycle - Lifecycle [module] running shutdown hook
```

Comment:
I find this output a good idea, even if it is long-winded. Could we add some basic commentary, though, as I'm not sure what refers to what. Things e.g. "here the shutdown is initiated", "this is the lifecycle prestop being triggered for the middle manager", "this is where Kubernetes issues a SIGKILL" etc.

Reply:
added the actual lifecycle stop output

Reply:
Thanks!